apps/meteor/server/services/moderation/service.ts

```diff
@@ -0,0 +1,74 @@
+import { ServiceClassInternal } from '@rocket.chat/core-services';
+import type { IModerationService } from '@rocket.chat/core-services';
+import { Permissions, Roles, Users } from '@rocket.chat/models';
+
+import { createOrUpdateProtectedRoleAsync } from '../../lib/roles/createOrUpdateProtectedRole';
+import { explorerPermissions as explorerExplicitPermissions, novicePermissions, trustRoles } from '../../../app/moderation/lib/permissions';
+import { Logger } from '../../lib/logger/Logger';
+
+export class ModerationService extends ServiceClassInternal implements IModerationService {
+	protected name = 'moderation';
+
+	private logger = new Logger('Moderation');
+
+	async removeTrustRoles(): Promise<void> {
+		const role = await Roles.findOneById(trustRoles[0], { projection: { _id: 1 } });
```

Comment:
Instead of `trustRoles[0]` why not making trustRoles something like an enum or so? That way you can use trustRoles.XXX instead of a hard coded number

Reply:
maybe .. yeah. at the end of the day it would check for any one roles, be it indexed or an enum.

i agree with Gazzo's recommendations though .. so I'll leave this one unresolved and wait for Devanshu to come back online 😬